jbmc/src/java_bytecode/java_bytecode_language.h

```diff
@@ -62,6 +63,10 @@ Author: Daniel Kroening, kroening@kroening.com
   "                              (byte, short, int, long, float, double) to an entry\n" /* NOLINT(*) */ \
   "                              point within the given range; lower bound L\n" /* NOLINT(*) */ \
   "                              and upper bound U must be integers\n" /* NOLINT(*) */ \
+  " --java-assume-inputs-integral\n" \
+  "                              only initialize numerical primitive-typed parameter\n" /* NOLINT(*) */ \
+  "                              to an entry point that is integral, not fractional;\n" /* NOLINT(*) */ \
```

Comment:
:pick: I would avoid using `fractional` here, as it's not the best way of describing floats (even if the can be seen as fractional from a mathematical point of view).
I would also avoid the notion of `entry-point`, as it can be confused with the entry-point of the analysis.

:no_entry_sign: Also, `parameter` is not quite sufficient, as this option also affects class fields.
I would simply write:
```
Force float and double inputs to have integer value.
```

Reply:
Done